www/class/centreonUtils.class.php

```diff
@@ -61,34 +61,6 @@ class CentreonUtils
      * Defines all self-closing html tags allowed
      */
     public static $selfclosingHtmlTagsAllowed = ['br', 'hr'];
-    
-    /**
-     * Converts Object into Array
-     *
-     * @param int $idPage
-     * @param boolean $redirect
-     * @return mixed
-     */
-    public function visit($idPage, $redirect = true)
```

Comment:
what was the purpose of this method ?

Reply:
seems to build a full url from a topology id and redirect to it
I've checked and it's not used in all centreon projects